### ISSUE TITLE
How much training data is generally required to fine-tune the model 

### ISSUE THREAD

**Issue by happyday521** [mazurowski-lab/finetune-SAM#8]
Hi，

I want to fine-tune the SAM mask decoder to only a single labeled dataset unsing supervised training (i.e., Setup 1 in your paper). 
Since I have no experience in this field, I would like to ask how much training data is generally required to fine-tune the model in this setting?

Thanks，look forward to your reply！

**Comment by haoyudong-97** [comment]
It depends on the dataset, but my educational guess is at least 100 cases if you only want to fine-tune the decoder. In the meantime, I suggest fine-tuning both the encoder and decoder since this strategy is almost always better than only fine-tuning the decoder.

**Comment by happyday521** [comment]
Get it. Thanks very much!
